root/src/main/resources/templates/style.css

```diff
@@ -0,0 +1,19 @@
+#site {
```

Comment:
Same here, seems a bit different. Maybe you were transferring the old code over?

Reply:
They are the same files. Prettier made some changes as well.

Reply:
Most up to date file is here: https://github.com/The-Movie-List-Makers/mymovielist/blob/main/style.css